CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,85 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
For old Unix users like me please use a line length of 78 characters.

Reply:
Done

Reply:
if we copy an existing CoC verbatim, perhaps should keep as is?